<launch>

    <remap from="/mementar/echo" to="/ActionRecognition/inputFacts"/>
    <group>
        <include file="$(find procedural)/launch/ontologenius_mementar.launch">
<!--            <arg name="name" value="$(arg onto_display)"/>-->
        </include>
    </group>


    <group>

        <param name="PATH_YAML" type="string"
               value="/home/avigne/Projets/Procedural/catkin_ws/src/Procedural/src/reader/test.yaml"/>
        <param name="TTL_BUFFER" type="double" value="25"/>
        <param name="MAX_SIZE_BUFFER" type="int" value="500"/>

        <arg name="name" default="PR2"/>

        <node name="recognition" pkg="procedural" type="recognition" output="screen" args="-n $(arg name)"/>
    </group>
</launch>